changelog/21.0/21.0.0/summary.md

```diff
@@ -24,6 +26,14 @@
     - **[vtctldclient ChangeTabletTags](#vtctldclient-changetablettags)**
     - **[Support for specifying expected primary in reparents](#reparents-expectedprimary)**
 
+## <a id="known-issue"/>Known Issue</a>
+
+### <a id="backup-reports-as-successful"/>Backup reports itself as successful despite failures</a>
+
+In this release, we identified an issue where a backup may succeed even if a file fails to be backed up.
+Leading to a successful backup, even if some errors occurred.
+This only happen with the Builtin Backup Engine, and when all files have already been initiated in the backup process.
```

Comment:
```suggestion
This issue exists only with the `builtin` backup engine, and it can occur only when the engine has already started backing up all files.
```

Reply:
Fixed via [1205315](https://github.com/vitessio/vitess/pull/17062/commits/1205315b507ad45d65b6f6651e5ab47eb1c267f1)